packages/ibm-products/src/components/Datagrid/Datagrid.docs-page.js

```diff
@@ -416,6 +416,30 @@ return <Datagrid datagridState={datagridState} />;
           `,
         },
       },
+      {
+        description: `Batch actions can be included by providing \`batchActions: true\` and \`toolbarBatchActions: CarbonButtonProps[]\` to the \`useDatagrid\` hook. While passing in a \`DatagridBatchActions\` component will also work it does not have some of the same responsive behavior built-in, thus it is not the recommended approach, as compared to using the \`batchActions\` and \`toolbarBatchActions\` properties.`,
```

Comment:
Small suggestion for clarity. I think it’s clearer at least. 🤔 

```suggestion
        description: `Batch actions can be included by providing \`batchActions: true\` and \`toolbarBatchActions: CarbonButtonProps[]\` to the \`useDatagrid\` hook. While passing in a \`DatagridBatchActions\` component will also work, it does not have the same responsive behavior built-in, thus we recommend using the \`batchActions\` and \`toolbarBatchActions\` properties when possible.`,
```